src/thor/isochrone.cc

```diff
@@ -83,8 +83,29 @@ void Isochrone::ConstructIsoTile(const bool multimodal, const unsigned int max_m
     max_distance = max_seconds * 70.0f * kMPHtoMetersPerSec;
   }
 
+  // Form bounding box that's just big enough to surround all of the locations.
+  PointLL center_ll = origin_locations[0].latlng_;
+  AABB2<PointLL> loc_bounds(center_ll.lng(), center_ll.lat(), center_ll.lng(), center_ll.lat());
+
+  for (const auto& loc : origin_locations) {
+    loc_bounds.Expand(loc.latlng_);
+  }
+
+  // Find the location closest to the center.
+  PointLL bounds_center = loc_bounds.Center();
```

Comment:
this is not the geometric center of the box? specifically which means the lat of this coord will be shifted away from the center as this box moves towards the poles. im not sure what effect that will have on isochrones but we can fix it by fixing `AABB2::Center` to use the midpoint algorithm of two diagonal corners. maybe it doesnt matter though